### PR TITLE
adds the W command to go to next WORD

### LINKED ISSUE #51: Add word navigation -- [numeral] WORDs forward
**Reproduction**: enter insert mode. enter "this is some text". use space+shift to reenter normal mode. use gg to go to beginning of line. use 'W'.

**Expected behavior**: cursor moves to the next WORD

**Actual behavior**: unknown command "W".

### PR DESCRIPTION
**Pull request by ducks**
Fixes #51

### CODE REVIEW

File: src/caret/movement.rs [hunk 1]
```diff
@@ -175,4 +175,24 @@ impl Editor {
 
         None
     }
+
+    /// Get next WORD forward
+    /// "A WORD consists of a sequence of non-blank characters, separated with
+    /// whitespace.  An empty line is also considered to be a WORD."
+    pub fn _next_word_forward(&self, n: usize) -> Option<usize> {
```
Comment (ticki) [inline src/caret/movement.rs:182]: Why `_`? I assume it is because it was unused? It isn't anymore.

Comment (ducks) [inline src/caret/movement.rs:182]: Damnit,  sorry, I meant to comment that up.

It is to show the difference in functions for WORD vs word.
I tried writing it as next_WORD_forward but rustc gave me a linting error.

Lemme know if you would like that updated and I can make the change.

Comment (ducks) [inline src/caret/movement.rs:182]: @ticki any thoughts on a better function naming?

Comment (ticki) [inline src/caret/movement.rs:182]: `next_word_forward` is fine.

Comment (ducks) [inline src/caret/movement.rs:182]: well this is for W which goes to the next WORD. w goes to next word.

we either need functions for both or make that one function do both.

**Comment by ticki** [comment]
Nice, thanks.

**Pull request merged**